### PR TITLE
[NT-0] feat: Update Example visionOS README & example code update

### PR DESCRIPTION
**Pull request by mag-richardzirbes**
connected-spaces-platform Pull Request

**For the reviewer**

* [ ] If required, are the changes covered by appropriate tests?
* [ ] Are any public-facing API changes well documented?
* [ ] Is the code easily readable and extensible and/or follow existing conventions?

### CODE REVIEW

File: Examples/Multiplayer/CSharp/Csp-Unity-VisionPro-Example/Assets/Scripts/HelloWorld.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public class HelloWorld : MonoBehaviour
     [SerializeField] private LocalPlayer localPlayerPrefab;
     [SerializeField] private RemotePlayer remotePlayerPrefab;
     
-    [SerializeField] private TextMeshProUGUI titleTenentReplaceLabel;
+    [SerializeField] private TextMeshProUGUI[] titleTenentReplaceLabels;
```
Comment (mag-richardzirbes) [inline Examples/Multiplayer/CSharp/Csp-Unity-VisionPro-Example/Assets/Scripts/HelloWorld.cs:24]: The sign in UI also had a location to substitute the Tenant ID.
Screenshot 2024-06-07 at 2 38 09 PM

File: Examples/Multiplayer/CSharp/Csp-Unity-VisionPro-Example/Assets/XR/Resources/XRSimulationRuntimeSettings.asset [hunk 2]
```diff
@@ -0,0 +1,67 @@
+%YAML 1.1
```
Comment (mag-richardzirbes) [inline Examples/Multiplayer/CSharp/Csp-Unity-VisionPro-Example/Assets/XR/Resources/XRSimulationRuntimeSettings.asset:1]: autogenerated file

File: Examples/Multiplayer/CSharp/Csp-Unity-VisionPro-Example/Assets/XR/UserSimulationSettings/Resources/XRSimulationPreferences.asset [hunk 3]
```diff
@@ -0,0 +1,17 @@
+%YAML 1.1
```
Comment (mag-richardzirbes) [inline Examples/Multiplayer/CSharp/Csp-Unity-VisionPro-Example/Assets/XR/UserSimulationSettings/Resources/XRSimulationPreferences.asset:1]: autogenerated file

**Comment by mag-richardzirbes** [comment]
You can view the README in it's entirety here:
https://github.com/magnopus-opensource/connected-spaces-platform/tree/feature/OPE-1700_example_vision_os_readme/Examples/Multiplayer/CSharp/Csp-Unity-VisionPro-Example

**Review by MAG-SamBirley — APPROVED** [review]
LGTM!

**Review by MAG-AdamThorn — APPROVED** [review]

**Pull request merged**